eng/doc/fips/README.md

```diff
@@ -117,9 +158,25 @@ To make the Go runtime panic during program initialization if FIPS mode is not e
 
 > Unlike `opensslcrypto`, a Windows program built with `cngcrypto` doesn't include the ability to enable/disable FIPS, only ensure it's enabled. Windows FIPS mode is not a per-process setting, and changing it may require elevated permissions. Adding this feature would likely have unintended consequences.
 
-## Usage: Extra runtime configuration options
+## Usage: Extra configuration options
+
+### Enable FIPS mode at compile time
```

Comment:
Enable or requires?

Reply:
I struggled with this header. 😄 I started with "Require FIPS mode at compile time", but I see that as saying the *compilation environment* is what should run in FIPS mode. "Enable FIPS mode" seemed a little better because it uses the terms set up in the previous section ("enable FIPS mode" could mean "GOFIPS=1"), but it's not really any good, seeing it again now. I think this fixes it:

```suggestion
### Build option to require FIPS mode
```